README.md

```diff
@@ -1,6 +1,6 @@
 # Structured Field Tests
 
-These are test cases for implementations of [Structured Fields for HTTP](http://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html). They currently test draft 17 of the specification.
```

Comment:
I'm not sure if this accurate to remove "They currently test draft 17 of the specification.".